integrations/api_repo_git_commits_test.go

```diff
@@ -132,3 +132,20 @@ func TestDownloadCommitDiffOrPatch(t *testing.T) {
 		resp.Body.String())
 
 }
+
+func TestGetFileHistory(t *testing.T) {
+	user := db.AssertExistsAndLoadBean(t, &models.User{ID: 2}).(*models.User)
+	// Login as User2.
+	session := loginUser(t, user.Name)
+	token := getTokenForLoggedInUser(t, session)
```

Comment:
Wasn't that made redundant with an extra function in one PR I reviewed? Or has that PR not been merged yet?

Reply:
I copied it from the other tests in this file, if there is a better way in main it's not used there